packages/code-studio/src/settings/SettingsMenu.tsx

```diff
@@ -177,7 +195,25 @@ export class SettingsMenu extends Component<
                 </a>
               </div>
               <div className="app-settings-footer-item">
-                <div className="text-muted">Version {version}</div>
+                <div className="text-muted">UI Version {version}</div>
+              </div>
+              <div className="app-settings-footer-item">
+                <div className="text-muted">
+                  Engine Version {deephavenVersion}{' '}
+                  <div className="detailed-server-config">
+                    <FontAwesomeIcon icon={vsInfo} />
+                    <Tooltip>
+                      <div className="detailed-server-config-info">
+                        {getRow('Deephaven Version     ', deephavenVersion)}
+                        {getRow('Java Version', javaVersion)}
+                        {getRow('Barrage Version', barrageVersion)}
+                        {/* <div>Deephaven Version {deephavenVersion}</div>
+                        <div>Java Version {javaVersion}</div>
+                        <div>Barrage Version {barrageVersion}</div> */}
```

Comment:
Delete the commented out code, and remove the unnecessary whitespace after Deephaven Version.
```suggestion
                        {getRow('Deephaven Version', deephavenVersion)}
                        {getRow('Java Version', javaVersion)}
                        {getRow('Barrage Version', barrageVersion)}
```